src/styles/stars.css

```diff
@@ -0,0 +1,18 @@
+.stars--4 {
```

Comment:
you've removed code you don't need for this task, but I'd keep it
why? for example, I see `stars--4`, but I don't know what it refers to (actually, to `stars`)
generally, such cases may confuse people looking at your code